install.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
I assume this is not required? since we have make targets that have a number of these things?

Reply:
not sure what you are asking here - you need the `#!/usr/bin/env` (could also use `#!/bin/bash`) in order for a shell script to be executable with the specified interpreter.